helm/examples/efs/README.md

```diff
@@ -96,8 +108,7 @@ Pick a VPC in the the same region of EFS server
   aws ec2 describe-vpcs --region <REGION>
 ```
 The output shows the VPC IDs.
-
-Get all the subnets of the VPC
+ll the subnets of the VPC
```

Comment:
> ll the subnets of the VPC

I assume this is supposed to still say "Get all the subnets of the VPC" ?

Reply:
Get all the subnets of the VPC

Reply:
Fixed